_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,84 @@
+
+This figure shows the abundance of each spacer sequence in the experiment, and each colour is a different day in the 14-day experiment. Each sequence is sorted from most common to least common (left to right), and the y-axis shows the frequency of each sequence. For most days, the most common sequence at rank 1 has a frequency of 10<sup>-1</sup> = 10%, and the least common sequences have frequencies around 10<sup>-5</sup>.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/banfield_data.svg" alt="banfield_data" width="400px" style="padding:5px;">
+</p>
+
+What can we learn from a spacer abundance distribution that looks like this? Does observing a broad spacer distribution tell us anything about the underlying process that created it? There are many reasons why some spacers might be more common than others. Bacteria might prefer to acquire certain spacer sequences over others, perhaps because of the way the phage genome is arranged or because of the type of proteins used for acquisition. Some spacers might be more effective than others at providing immunity, and these sequences might grow more common over time because of natural selection. Or perhaps there's nothing special about particular sequences, and it's some other random process that causes some to be common and others to be rare. There are two big questions here: first, can any of these processes produce a broad spacer abundance distribution? And second, if we see a distribution like this, can we draw any conclusions about what's going on underneath? Does process A cause result B, and is result B evidence of process A? These might seem like the same question, but we'll see that there's a subtle difference.
+
+Experiments can chip away at the first question. In the original study, they found in addition to this broad distribution that certain regions of the phage genome were more likely to be a source for spacers. Based on this, they concluded that the high-abundance spacers were special: their sequence makes them more likely to be high abundance. That's one piece of evidence that spacer sequences matter, and it's supported by other experiments. Here’s a figure from another experiment that looked specifically at the process of spacer acquisition - they infected a population of bacteria with phages, then sequenced new spacers 24 hours later without leaving the bacteria lots of time to grow and die. They performed the same experiment twice (two 'replicates'), and this plot shows the frequency of spacer sequences in the first replicate versus the frequency of the same spacer sequences in the second replicate. The data points are strongly correlated: they fall on or near a straight diagonal line with slope 1. This means that if a spacer sequence was highly abundant in the first experiment, it is also generally highly abundant in the second experiment, suggesting that there is indeed something about the spacer itself that destines it for high or low abundance[^3]. 
```

Comment:
I think this looks like a tough paragraph but you've written it well. A does cause B.